New package: LMDiskANN v1.1.0

JuliaRegistrator opened the pull request:
- Registering package: LMDiskANN
- Repository: https://github.com/mantzaris/LMDiskANN.jl
- Created by: @mantzaris
- Version: v1.1.0
- Commit: 823cfb0891230bb3b8fb11476753fe464ef13213
- Reviewed by: @mantzaris
- Reference: https://github.com/mantzaris/LMDiskANN.jl/commit/823cfb0891230bb3b8fb11476753fe464ef13213#commitcomment-159988053
- Description: Julia Implementation of Low Memory Disk ANN (LM-DiskANN)
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6b71a6c46f70d813c5907d1d9f194923639c739cf4be3183c69105c2aaf1dfb6115da63abaec51bb1c42ced1225d03236ff1786120a13fae5b220388337a37ae3d15cde306704c559edc13d4c56faa7ca513a82f4122d37c544ddd35a4cd94b061edf058fdba04a34b4e2de269e54f785e3045bda5aa8e69f93e2b944739d3698c2d7530b5546c7a2e6acd286d76b29202903632f89b457482886c9bf6579bb966f86a0db841c4f82f5469437e22d86d3 -->

goerz commented:
When fixing automerge issues, please keep the version number the same. Also, [you are using TagBot](https://github.com/mantzaris/LMDiskANN.jl/blob/main/.github/workflows/TagBot.yml), which means you should let TagBot create the tag for a release _after_ that release has made it into the registry.

You may find the discussion in https://github.com/JuliaRegistries/General/pull/132126#issuecomment-2931237783 relevant.

In this case, I understand that you may have to coordinate this with JOSS, so probably an initial version `v1.1.0` is appropriate, but you may want to roll back `v1.2.0` in https://github.com/JuliaRegistries/General/pull/132816.

goerz commented:
My recommendation would be:

1. Delete the tag [v1.2.0](https://github.com/mantzaris/LMDiskANN.jl/releases/tag/v1.2.0) (this, you should do anyway).

2. Edit your [Project.toml](https://github.com/mantzaris/LMDiskANN.jl/blob/main/Project.toml) to have `version = "1.1.0"` and create a commit for that.

3. In the GitHub UI, find the commit you just created and comment `@JuliaRegistrator register` on it (cf. the [JuliaRegistrator instructions](https://github.com/JuliaRegistries/Registrator.jl?tab=readme-ov-file#via-the-github-app)). This will update this existing registration PR. You can include [release notes in the call to JuliaRegistrator](https://github.com/JuliaRegistries/Registrator.jl?tab=readme-ov-file#release-notes), like the note

   > Corrections and modifications at the request of the editor and reviewers of JOSS (Journal of open source software). Various improvements have been made over the previous version v.1.0.0.

   that you had alongside your tag.

The alternative would be to only do step (1) (delete the tag), and then we'd just let https://github.com/JuliaRegistries/General/pull/132816 automerge as `v1.2.0`. The consequence of that would be that you'd be skipping `v1.1.0`. If I was you, that gap would bother me, and it could be confusing for your users ("what happened to v1.1?").

mantzaris commented:
> My recommendation would be: ...
> The alternative would be to only do step (1) (delete the tag), and then we'd just let https://github.com/JuliaRegistries/General/pull/132816 automerge as v1.2.0. The consequence of that would be that you'd be skipping v1.1.0. If I was you, that gap would bother me, and it could be confusing for your users ("what happened to v1.1?").

Ok, I will work on it now. I agree with what you say about that skipping of v1.1.0 being missing, but I made a mistake in the publication for JOSS journal and even the doi software url that says v1.2.0. So I am not sure what is a better choice of the 2. If you think going to v1.1.0 is the best avenue, I will take that but I am thinking v1.2.0 cause of the journal, but maybe that is not a problem?

JuliaRegistrator committed:
New package: LMDiskANN v1.1.0

UUID: a2e5363b-c4e1-4593-9e2f-a2937b345987
Repo: https://github.com/mantzaris/LMDiskANN.jl.git
Tree: 9d8a6f7cd6978f28fd6392d8b33ab922dcf9d9c6

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
